Devscord.DiscordFramework/Framework/Server.cs

```diff
@@ -11,16 +11,16 @@ namespace Devscord.DiscordFramework.Framework
 {
     public static class ServerInitializer
     {
-        public static bool Initialized = false;
+        private static bool _initialized = false;
```

Comment:
Myśle że to powinna być opcja możliwa do podejrzenia z zewnątrz
Ale obecnie nie ma to dużego znaczenia, może zostać tak jak jest

Reply:
```suggestion
        private static bool Initialized { get; private set; } = false;
```